helmfile/values/harbor.yaml.gotmpl

```diff
@@ -73,6 +75,9 @@ externalURL: https://harbor.{{ .Values.global.baseDomain }}
 
 harborAdminPassword: {{ .Values.harbor.password }}
 
+clair:
+  enabled: false
```

Comment:
Did both trivy and clair run before this?

Reply:
Yes